mysql-compatibility.md

```diff
@@ -45,7 +45,7 @@ However, some features of MySQL are not supported. This could be because there i
 
 ### Auto-increment ID
 
-+ In TiDB, auto-increment columns are only guaranteed to be unique and incremental on a single TiDB server, but they are *not* guaranteed to be incremental among multiple TiDB servers or allocated sequentially. It is recommended that you do not mix default values and custom values. Otherwise, you might encounter the `Duplicated Error` error message.
++ In TiDB, auto-incremental columns are globally unique, but only guaranteed to be incremental and sequential on a single TiDB server. They are *not* guaranteed to be incremental among multiple TiDB servers or allocated sequentially. It is recommended that you do not mix default values and custom values. Otherwise, you might encounter the `Duplicated Error` error message.
```

Comment:
```suggestion
+ In TiDB, auto-incremental columns are globally unique and incremental on a single TiDB server. They are *not* necessarily incremental and sequential among multiple TiDB servers. It is recommended that you do not mix default values and custom values. Otherwise, you might encounter the `Duplicated Error` error message.
```
原文是 “TiDB 的自增列仅保证唯一，也能保证在单个 TiDB server 中自增，但不保证多个 TiDB server 中自增，不保证自动分配的值的连续性”。

Reply:
@shichun-0415 The original Chinese version also needs to be updated. See https://github.com/pingcap/docs-cn/pull/8604/files.